.gitignore

```diff
@@ -5,7 +5,6 @@
 /build
 /captures
 .externalNativeBuild
-/gradle.properties
```

Comment:
https://github.com/k0shk0sh/FastHub/blob/d2b02cc78d96a8ad8af74efddf2020c014e3c2c8/debug_gradle.properties#L1

Add it here